mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -338,8 +338,10 @@ function render_global_header( $attributes = array() ) {
 
 	// Preload the menu font.
 	if ( is_callable( 'global_fonts_preload' ) ) {
-		if ( ! is_rosetta_site() ) {
-			global_fonts_preload( 'Inter latin' );
+		/* translators: Subsets can be any of cyrillic, cyrillic-ext, greek, greek-ext, vietnamese, latin, latin-ext. */
+		$subsets = _x( 'latin', 'Global menu font subsets, comma separated', 'wporg' );
+		foreach ( (array) $subsets as $subset ) {
```

Comment:
This `$subsets` value will always be a string, converted to an array of length 1, even if the translator specifies `cyrillic, cyrillic-ext`. You can `explode( ',', $subsets );` to split it into the subset parts. Or maybe `global_fonts_preload` should take a second parameter which is the comma-separated subsets, and `preload_font` can handle splitting apart the subsets?

Reply:
OH! I only tested with:
```
$subsets = 'cyrillic, cyrillic-ext';
foreach ( (subsets) $ as $subset ) {
    echo $subset;
} 
```
And it prints out `cyrillic, cyrillic-ext`. 
Apparently the structure is:
```
Array
(
    [0] => cyrillic, cyrillic-ext
)
```
Thanks for pointing it out!